distribution/scripts/docker/install-docker.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019, WSO2 Inc. (http://wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+# Installation script for setting up docker on Linux.
+# ----------------------------------------------------------------------------
+
+default_user=""
+if [[ ! -z $SUDO_USER ]]; then
+    default_user="$SUDO_USER"
+else
+    default_user="ubuntu"
+fi
+user="$default_user"
```

Comment:
Use this user in following command: `sudo usermod -aG docker $user`. Do this after docker installation. See also https://docs.docker.com/install/linux/docker-ce/ubuntu/